### PR TITLE
Update ostree-ng.sh to support auto pushing to container registry

### PR DESCRIPTION
**Pull request by mcattamoredhat**
PR to update ostree-ng.sh to support auto pushing to container registry.

### CODE REVIEW

File: ostree-ng.sh [hunk 1]
```diff
@@ -406,10 +431,18 @@ greenprint "🧹 Clean up compose"
 sudo composer-cli compose delete "${COMPOSE_ID}" > /dev/null
 sudo composer-cli blueprints delete container > /dev/null
 
-# Remove tag from quay.io repo
 greenprint "Remove tag from quay.io repo"
 skopeo delete --creds "${QUAY_USERNAME}:${QUAY_PASSWORD}" "${QUAY_REPO_URL}:${QUAY_REPO_TAG}"
 
+# # Remove tag from quay.io repo
```
Comment (henrywang) [inline ostree-ng.sh:437]: Let's remove the comment out code if we do not need them. Thanks.

**Review by henrywang — CHANGES_REQUESTED** [review]

File: ostree-ng.sh (hunk 1 above)
Comment (mcattamoredhat) [reply ostree-ng.sh]: Ok, done! Thanks!

**Comment by henrywang** [comment]
CS9 failed test is caused by issue #155

**Review by henrywang — APPROVED** [review]
Well done! Thanks @mcattamoredhat for doing this.

**Pull request merged**